interface/wx/graphics.h

```diff
@@ -391,6 +391,8 @@ class wxGraphicsBitmap : public wxGraphicsObject
     results visible you need to update the content of the context
     by calling wxGraphicsContext::Flush() or by destroying the context.
 
+    @remarks Logical functions does not work with wxGraphicsContext.
```

Comment:
Apart from the grammar, IIRC some logical functions do work with GC, even if just a few of them and probably not the same ones on different platforms. i.e. `wxCOPY` probably works for all. This can be easily checked in `drawing` sample.
Also, did you test the change? Is it ok to have 2 `@remarks` tags for the same target?